rust/src/push/evaluator.rs

```diff
@@ -171,7 +171,7 @@ impl PushRuleEvaluator {
                     Ok(true) => {}
                     Ok(false) => continue 'outer,
                     Err(err) => {
-                        warn!("Condition match failed {err}");
+                        warn!(target: "synapse.synapse_rust","Condition match failed {err}");
```

Comment:
wonder if there's something less intensive than having to write `target: ...` everywhere.

Reply:
I couldn't find a way to set this at like a module / library level, but happy to be proved wrong!

Reply:
fair enough. If we get desperate we could possibly make a wrapper macro that looks at the current source file but that's for later if ever :)

Reply:
Well apparently I was wrong and it does work fine, see https://github.com/matrix-org/synapse/pull/14976#issuecomment-1415817720